tests/src/JIT/jit64/opt/cse/hugeexpr1.csproj

```diff
@@ -30,6 +30,9 @@
     <DebugType>Full</DebugType>
     <Optimize>True</Optimize>
   </PropertyGroup>
+  <PropertyGroup>
+    <JitOptimizationSensitive>true</JitOptimizationSensitive>
```

Comment:
Yes, but that assumes that the build architecture matches the execution architecture. This has caused confusion in the past since we don't support end-to-end test build+run on all platforms.  Like #6680.  In this case, you might get away with it if people only use x86 builds to test 32-bit platforms, but the reward doesn't justify the risk to me.  Do you still think it's worth changing?